src/pages/Fortune/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 import Wheel from '../../components/Wheel';
 
-import './index.css';
+import '../../styles/index.css'
```

Comment:
why do you need to import common styles inside all of your components? won't it better to import it once inside index.jsx ?

Reply:
Update in next commit